daemon/openrazer_daemon/daemon.py

```diff
@@ -445,6 +449,14 @@ def _add_device(self, device):
                     self.device_added()
                 else:
                     logging.warning("Could not get serial for device {0}. Skipping".format(sys_name))
+            else:
+                # Basically find the other usb interfaces
+                device_match = sys_name.split('.')[0]
+                for d in self._razer_devices:
+                    if device_match in d.device_id and d.device_id != sys_name:
+                        if not sys_path in d.dbus.additional_interfaces:
+                            d.dbus.additional_interfaces.append(sys_path)
+                            return
```

Comment:
instead of splitting on the sysname wouldn't it be better to use udev to run up the hierarchy and find devices with the same parent?

Reply:
At last in pyudev taking the device and calling `.parent` seems to result in the usb port node being shown. I am not too familiar with udev or USB though.